feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.html

```diff
@@ -0,0 +1,128 @@
+<ng-container *ngIf="product$ | async as product">
+  <div
+    class="cx-product-card"
+    [ngClass]="{
+      'cx-product-card-selected': isProductCardSelected()
+    }"
+  >
+    <div class="cx-product-card-rows">
+      <div class="cx-product-card-imgs">
+        <cx-media
+          [container]="product?.images?.PRIMARY"
+          format="product"
+        ></cx-media>
+      </div>
+
+      <div class="cx-product-card-info">
+        <div class="cx-product-card-name">
+          <p>
+            {{ product.name }}
+          </p>
+        </div>
+        <div class="cx-product-card-code" *ngIf="product.code">
+          {{ 'configurator.attribute.id' | cxTranslate }}:
+          {{ product.code }}
+        </div>
+        <cx-configurator-show-more
```

Comment:
Does `ProductScope.LIST` include `description`?

Reply:
Picked

Reply:
No, it doesn't. We have to change to 'DETAILS'.

Reply:
Done

Reply:
@Platonn Good hint. Agree, created a specific scope that only has pictures and description, and used that in productCard

Reply:
@Changsuwan FYI